### ISSUE TITLE
@contexts no longer show up if I type the @ sign

### ISSUE THREAD

**Issue by priyashpatil** [continuedev/continue#2968]
### Before submitting your bug report

- [X] I believe this is a bug. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue) that reports the same bug
- [X] I've seen the [troubleshooting guide](https://docs.continue.dev/troubleshooting) on the Continue Docs

### Relevant environment info

```Markdown
Version: 1.95.3
Commit: f1a4fb101478ce6ec82fe9627c43efbf9e98c813
Date: 2024-11-13T14:50:04.152Z
Electron: 32.2.1
ElectronBuildId: 10427718
Chromium: 128.0.6613.186
Node.js: 20.18.0
V8: 12.8.374.38-electron.0
OS: Darwin arm64 24.1.0
continue v0.9.231 (pre-release)
```

### Description

The context dropdown does not work.

Reference: https://discord.com/channels/1108621136150929458/1306627295967973387

**Comment by vlebert** [comment]
Same here :(

**Comment by Patrick-Erichsen** [comment]
Thanks for the heads up Priyash! Going to keep this open but will be following up in this thread: https://github.com/continuedev/continue/issues/2990

**Comment by Patrick-Erichsen** [comment]
Hi folks, this should be fixed here: https://github.com/continuedev/continue/pull/3086

Will be available in pre-release v0.9.236 by end of day. Please let us know if you're still having issues!
